src/main/kotlin/solve/importer/view/LoadingScreen.kt

```diff
@@ -1,42 +1,63 @@
 package solve.importer.view
 
+import io.github.palexdev.materialfx.controls.MFXProgressSpinner
 import javafx.geometry.Insets
-import javafx.geometry.Pos
-import javafx.scene.control.ContentDisplay
-import javafx.scene.image.ImageView
 import javafx.scene.layout.BorderPane
-import javafx.scene.text.Font
+import javafx.scene.layout.Priority
+import javafx.scene.layout.VBox
+import javafx.scene.paint.Color
 import javafx.stage.Stage
 import kotlinx.coroutines.cancel
-import solve.constants.IconsImporterLoadingPath
-import solve.utils.loadResourcesImage
+import solve.styles.Style
+import solve.utils.mfxButton
 import tornadofx.*
 
 class LoadingScreen : View("Loading") {
-    private val loading = loadResourcesImage(IconsImporterLoadingPath)
+    private val primaryColor: Color = Color.valueOf(Style.primaryColor)
+
+    private val progressSpinner = MFXProgressSpinner().apply {
+        color1 = primaryColor
+        color2 = primaryColor
+        color3 = primaryColor
+        color4 = primaryColor
+    }
 
     override fun onDock() {
         (root.scene.window as Stage).minWidth = 390.0
     }
 
     override val root =
         borderpane {
-            padding = Insets(0.0, 0.0, 8.0, 0.0)
+            prefHeight = 570.0
+            prefWidth = 453.0
+            style = "-fx-background-color: #ffffff"
+            top{
+                label("Import a directory") {
+                    hgrow = Priority.ALWAYS
+                    prefHeight=0.0
+                    prefWidth=200.0
+                    style = "-fx-font-family: ${Style.fontCondensed}; -fx-font-size: ${Style.headerFontSize}"
+                    BorderPane.setMargin(this, Insets(0.0, 0.0, 0.0, 24.0))
+                }
+            }
             center {
-                label {
-                    text = "Please, wait..."
-                    font = Font.font(20.0)
-                    graphic = ImageView(loading).apply {
-                        fitHeight = 80.0
-                        fitWidth = 80.0
+
+                vbox(15) {
+                    BorderPane.setMargin(this, Insets(200.0, 0.0,0.0,170.0))
+                    add(progressSpinner.apply {
+                        VBox.setMargin(this, Insets(0.0,0.0,0.0,30.0))
+                    })
+                    label("Please, wait...") {
+                        style = "-fx-font-size: ${Style.headerFontSize}; -fx-font-style: ${Style.fontCondensed}"
                     }
-                    contentDisplay = ContentDisplay.TOP
                 }
             }
             bottom {
-                button("Cancel") {
-                    BorderPane.setAlignment(this, Pos.BOTTOM_CENTER)
```

Comment:
Wrong position for the button

![image](https://user-images.githubusercontent.com/55746901/233854520-173f3088-ee4e-49fc-9058-518b9570872a.png)